src/components/widgets/support-case-widget.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Table,
+  TableVariant,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from '@patternfly/react-table';
+import { Button } from '@patternfly/react-core/dist/dynamic/components/Button';
+import { EmptyState } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateBody } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateIcon } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateVariant } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { Stack } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { StackItem } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { Title } from '@patternfly/react-core/dist/dynamic/components/Title';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/dynamic/icons/external-link-alt-icon';
+import { Label } from '@patternfly/react-core/dist/dynamic/components/Label';
+import HeadsetIcon from '@patternfly/react-icons/dist/dynamic/icons/headset-icon';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import SkeletonTable from '@patternfly/react-component-groups/dist/dynamic/SkeletonTable';
+
+export type Case = {
+  id: string;
+  caseNumber: string;
+  summary: string;
+  lastModifiedById: string;
+  severity: string;
+  status: string;
+};
+
+const SupportCaseWidget: React.FunctionComponent = () => {
+  const [cases, setCases] = useState<Case[]>([]);
+  const MAX_ROWS = 5;
+  const chrome = useChrome();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const columnNames = {
+    caseId: 'Case ID',
+    issueSummary: 'Issue summary',
+    modifiedBy: 'Modified by',
+    severity: 'Severity',
+    status: 'Status',
+  };
+
+  const fetchSupportCases = async () => {
+    const token = await chrome.auth.getToken();
+    const user = await chrome.auth.getUser();
+    const url = 'https://api.access.redhat.com/support/v1/cases/filter';
```

Comment:
Can we move this from the component to a const so we don't have to re-assign it on every fetch?